test/dynamo/test_functions.py

```diff
@@ -2378,14 +2378,53 @@ def fn(inputs):
         input2 = torch.arange(2, dtype=torch.bfloat16)
         inputs = [input1, input2]
 
-        opt_fn = torch.compile()(fn)
+        opt_fn = torch.compile()(fn)  # will graph break
         self.assertEqual(opt_fn(inputs), fn(inputs))
 
         torch._dynamo.reset()
 
-        with self.assertRaises(torch._dynamo.exc.Unsupported):
-            opt_fn = torch.compile(fullgraph=True)(fn)
-            opt_fn(inputs)
+        opt_fn = torch.compile()(fn)  # will graph break
+        self.assertEqual(opt_fn(inputs), fn(inputs))
```

Comment:
will graph break due to control flow

Reply:
Should revert this change - we should be ensuring we graph break here.